tests/serialisation_tests/test_serialisation.py

```diff
@@ -0,0 +1,314 @@
+    assert list(test_serialised_data["OutcomeData"][0].keys()) == [
+        "SubmissionID",
+        "ProgrammeID",
+        "ProjectID",
+        "StartDate",
+        "EndDate",
+        "Outcome",
+        "UnitofMeasurement",
+        "GeographyIndicator",
+        "Amount",
+        "ActualOrForecast",
+        "SpecifyIfYouAreAbleToProvideThisMetricAtAHigherFrequencyLevelThanAnnually",
+        "ProjectName",
+        "Place",
+        "OrganisationName",
+    ]
+    assert list(test_serialised_data["RiskRegister"][0].keys()) == [
+        "SubmissionID",
+        "ProgrammeID",
+        "ProjectID",
+        "RiskName",
+        "RiskCategory",
+        "ShortDescription",
+        "FullDescription",
+        "Consequences",
+        "PreMitigatedImpact",
+        "PreMitigatedLikelihood",
+        "Mitigations",
+        "PostMitigatedImpact",
+        "PostMitigatedLikelihood",
+        "Proximity",
+        "RiskOwnerRole",
+        "ProjectName",
+        "Place",
+        "OrganisationName",
+    ]
+
+    # check a couple of tables that all results are returned
+    assert len(test_serialised_data["RiskRegister"]) == len(RiskRegister.query.all()) == 27
+    assert len(test_serialised_data["ProjectDetails"]) == len(Project.query.all()) == 12
+
+
+def test_serialise_download_data_organisation_filter(seeded_test_client, additional_test_data):
+    """Assert filter applied reduces results returned."""
+    organisation = additional_test_data[0]
+    organisation_uuids = [organisation.id]
+    test_query_org = download_data_base_query(organisation_uuids=organisation_uuids)
+    test_serialised_data = serialise_download_data(test_query_org)
+
+    assert len(test_serialised_data["OrganisationRef"]) == 1
+    assert len(test_serialised_data["OrganisationRef"]) < len(Organisation.query.all())
+
+    assert len(test_serialised_data["ProjectDetails"]) == 4
+    assert len(test_serialised_data["ProjectDetails"]) < len(Project.query.all())
+
+
+def test_serialise_data_region_filter(seeded_test_client, additional_test_data):
+    """
+    when ITL region is passed, projects should be filtered by ITL region and any parent programmes with entirely
+    filtered out child projects should not be returned.
+    """
+    itl_regions = {ITLRegion.SouthWest}
+    test_query_region = download_data_base_query(itl_regions=itl_regions)
+
+    test_serialised_data = serialise_download_data(test_query_region)
+
+    #  read into pandas for ease of inspection
+    test_fund_filtered_df = pd.DataFrame.from_records(test_serialised_data["ProjectDetails"])
+
+    project4 = additional_test_data[7]
+    assert project4.project_id not in test_fund_filtered_df["ProjectID"]  # not in SW region
+    assert all(
+        ITLRegion.SouthWest in project.itl_regions
+        for project in Project.query.filter(Project.project_id.in_(test_fund_filtered_df["ProjectID"]))
+    )
```

Comment:
do we want some more tests like this for the other filters or happy with the queries tests?

Reply:
It basically does the same thing as queries tests, so no point re-inventing the wheel. My thoughts were that queries are tested in test_queries, serialisation is tested in test_serialisation. I started combining both here, but realised there was just a lot of duplication / testing the same thing twice. 

If I find any bugs when testing the download, I've generally been adding query/serialisation combination tests then, otherwise just assuming the individual test coverages on each is enough. Otherwise I'd be writing tests all month :D

Reply:
yeah it makes sense to do most of the testing in queries just wanted to make sure